defaults/main.yml

```diff
@@ -37,6 +37,9 @@ kafka_hosts:
 # Switch to enable topic deletion or not, default value is false
 kafka_delete_topic_enable: false
 
+# Switch to enable auto create topic or not, default value is true
+kafka_auto_create_topics: 'true'
```

Comment:
Shouldn't be true (without quotes)?

Reply:
This is because i don't want ansible to make the type conversion :|